### PR TITLE
merging for contact page

### PR DESCRIPTION
**Pull request by Aleumkim**

### CODE REVIEW

File: src/components/Contact/ContactList.js [hunk 1]
```diff
@@ -39,31 +70,91 @@ const ContactList = () => {
           setContacts(res.data);
         });
 }
+  const addContact= () => {
+    axios.post(`http://localhost:5003/api/contacts/${params.userId}`, contact)
```
Comment (smilecana) [inline src/components/Contact/ContactList.js:74]: don't use direct URL.

File: src/components/Contact/ContactList.js [hunk 2]
```diff
@@ -39,31 +70,91 @@ const ContactList = () => {
...
+
+  }
+  const deleteContact = (id) => {
+    axios.delete(`http://localhost:5003/api/contacts/${id}`)
+        .then(res => {
+          setAlert({...alert, open: true, message: res.data.message, status: res.status});
+          handleAlertModalClose();
+        })
+}
+const resetContact = () => {
```
Comment (smilecana) [inline src/components/Contact/ContactList.js:95]: please check the indentation.

**Review by smilecana — APPROVED** [review]

**Comment by yhupaul** [comment]
good job!!

**Review by yhupaul — APPROVED** [review]
good job!!

**Pull request merged**